packages/conf-qt/conf-qt.5.2.1/opam

```diff
@@ -28,20 +28,17 @@ post-messages:
     {os = "macos"}
 depexts: [
   ["qt5"] {os = "macos" & os-distribution = "homebrew"}
-  ["qt5-default" "qtdeclarative5-dev" "qt5-qmake"]
-    {os-distribution = "ubuntu"}
-  ["qt5-default" "qtdeclarative5-dev" "qt5-qmake"]
-    {os-distribution = "debian"}
-
-  # to set right paths for qt5 automatically we should 
```

Comment:
Comments have been removed here. Could you put them back?

Reply:
Also changing the indentation as less as possible would be a bit better to review